src/components/activity/ActivityDirectivesTable.svelte

```diff
@@ -117,6 +130,29 @@
     completeColumnDefs = [activityErrorColumnDef, ...(columnDefs ?? []), activityActionColumnDef];
   }
 
+  onMount(() => {
+    document.addEventListener(`keydown`, onKeyDown);
+  });
+
+  onDestroy(() => {
+    document.removeEventListener(`keydown`, onKeyDown);
+  });
+
+  function onKeyDown(event: KeyboardEvent) {
+    if (isMetaOrCtrlPressed(event)) {
```

Comment:
Can we also detect if the event is coming from an input so that we don't accidentally copy/paste when a user is trying to copy/paste text?